### PR TITLE
Use DownloadHeader from asl/components

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: client/components/download-header.js [hunk 1]
```diff
@@ -1,80 +0,0 @@
...
-  const props = useSelector(selector, shallowEqual);
-  const {
-    id,
-    title,
-    isGranted,
-    drafting,
-  } = props;
-  let { basename = window.location.href } = props;
-
-  basename = basename.replace(/\/edit/, '');
```
Comment (wheelsandcogs) [inline client/components/download-header.js]: I think this is missing from the asl/components version - I guess we need it?

Comment (wheelsandcogs) [reply client/components/download-header.js]: Sorted here https://github.com/UKHomeOffice/asl-projects/pull/367/files#diff-aa416c553720e93e8852480dcca1b545R99

File: client/project-router.js [hunk 2]
```diff
@@ -92,10 +96,12 @@ const ProjectRouter = () => {
     };
   });
 
+  const downloadBasename = basename.replace(/\/edit/, '');
```
Comment (lennym) [inline client/project-router.js]: This is a _very_ liberal regex. Does that have the potential to screw future us? Should we at least make sure it appears at the end?

Comment (lennym) [reply client/project-router.js]:
```suggestion
  const downloadBasename = basename.replace(/\/edit($|\/)/, '');
```

**Pull request merged**